### ISSUE TITLE
Error installing

### ISSUE THREAD

**Issue by WestMichigan** [oyvindkinsey/avionmqtt#1]
Building wheels for collected packages: bluepy
  Building wheel for bluepy (pyproject.toml) ... error
  error: subprocess-exited-with-error
  
  × Building wheel for bluepy (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [51 lines of output]
      /tmp/pip-build-env-fchk0ljn/overlay/lib/python3.12/site-packages/setuptools/dist.py:488: SetuptoolsDeprecationWarning: Invalid dash-separated options
      !!
      
              ********************************************************************************
              Usage of dash-separated 'description-file' will not be supported in future
              versions. Please use the underscore name 'description_file' instead.
      
              By 2025-Mar-03, you need to update your project and remove deprecated calls
              or your builds will no longer be supported.
      
              See https://setuptools.pypa.io/en/latest/userguide/declarative_config.html for details.
              ********************************************************************************
      
      !!
        opt = self.warn_dash_deprecation(opt, section)
      /usr/lib/python3.12/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py:251: _IncompatibleBdistWheel: wheel.bdist_wheel is deprecated, please import it from setuptools
      !!
      
              ********************************************************************************
              Ensure that any custom bdist_wheel implementation is a subclass of
              setuptools.command.bdist_wheel.bdist_wheel.
      
              By 2025-Oct-15, you need to update your project and remove deprecated calls
              or your builds will no longer be supported.
      
              See https://github.com/pypa/wheel/pull/631 for details.
              ********************************************************************************
      
      !!
        return _build_backend().build_wheel(wheel_directory, config_settings,
      /tmp/pip-build-env-fchk0ljn/overlay/lib/python3.12/site-packages/setuptools/dist.py:488: SetuptoolsDeprecationWarning: Invalid dash-separated options
      !!
      
              ********************************************************************************
              Usage of dash-separated 'description-file' will not be supported in future
              versions. Please use the underscore name 'description_file' instead.
      
              By 2025-Mar-03, you need to update your project and remove deprecated calls
              or your builds will no longer be supported.
      
              See https://setuptools.pypa.io/en/latest/userguide/declarative_config.html for details.
              ********************************************************************************
      
      !!
        opt = self.warn_dash_deprecation(opt, section)
      running bdist_wheel
      running build
      running build_py
      Working dir is /tmp/pip-install-2o4hibek/bluepy_afb83cd9c95f4859b9ec67411018de18
      execute make -C ./bluepy clean
      error: [Errno 2] No such file or directory: 'make'
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for bluepy
Failed to build bluepy
ERROR: Could not build wheels for bluepy, which is required to install pyproject.toml-based projects

**Comment by oyvindkinsey** [comment]
Yeah, that's an issue with the blueby install process - if I remember correctly, you need to install libglib separately, and then it will work.

  sudo apt-get install libglib2.0-dev

**Comment by oyvindkinsey** [comment]
@WestMichigan , did you get it working?

**Comment by oyvindkinsey** [comment]
You can now try the install script from https://github.com/oyvindkinsey/avionmqtt/blob/main/resources/Running%20as%20a%20service.md

Please feel free to reopen if you're still having problems.
